block.go

```diff
@@ -480,14 +480,15 @@ func (pb *Block) CleanTombstones(dest string, c Compactor) (bool, error) {
 	})
 
 	if numStones == 0 {
-		return false, nil
+		return false, [16]byte{}, nil
```

Comment:
any reason why you return `[16]byte{}` instead of `ulid.ULID{}`?

Reply:
Oops, didn't think about that. Changed.